framework/db/QueryBuilder.php

```diff
@@ -1151,6 +1151,9 @@ public function buildColumns($columns)
             }
 
             $columns = preg_split('/\s*,\s*/', $columns, -1, PREG_SPLIT_NO_EMPTY);
+            if ($columns === false) {
+                throw new InvalidParamException("$columns is not valid columns.");
```

Comment:
We know it's `false`. Why building a string from a variable?

Reply:
I am sorry, is fixed now.